docs/tutorials/advanced/rule_based_profilers/how-to-create-a-new-expectation-suite-using-rule-based-profilers.md

```diff
@@ -0,0 +1,130 @@
+---
+title: How to create a new Expectation Suite using Rule Based Profilers
+---
+import Prerequisites from '../../../guides/connecting_to_your_data/components/prerequisites.jsx';
+
+<Prerequisites>
+
+- Have configured a Datasource to connect to your data
+- Have a basic understanding of [Metrics in Great Expectations](https://docs.greatexpectations.io/en/latest/reference/core_concepts/metrics.html)
+- Have a basic understanding of [Expectation Configurations in Great Expectations](https://docs.greatexpectations.io/en/latest/reference/core_concepts/expectations/expectations.html#expectation-concepts-domain-and-success-keys)
+- Have read the sections in Core Concepts on [Profiling](../../../reference/core-concepts#profiling) and [Rule-based Profilers](../../../reference/core-concepts#rule-based-profilers)
+
+</Prerequisites>
+
+### Intro
+
+In this tutorial, you will develop hands-on experience with configuring a Rule-based Profiler to create an Expectation Suite. You will Profile several batches of NYC yellow taxi trip data to come up with reasonable estimates for the ranges of expectations for several numeric columns.
+
+### Set-up
+#### 1. Create a new Great Expectations project
```

Comment:
Please change all numbered Steps headers from H4 to H3 to allow them to display in the inline TOC.